### ISSUE TITLE
Suggestion: Feature Filter projects

### ISSUE THREAD

**Issue by awaissahmed80** [bobmagicii/vscode-dashboard#11]
Hi, is it possible for you  add search box at the top of main screen to filter projects and groups by keywords?

**Comment by bobmagicii** [comment]
yeah, sure. my personal dashboard also is getting kind of full and i end up getting lost. a search would be nice.
